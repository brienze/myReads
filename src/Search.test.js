import React from 'react';
import {shallow} from 'enzyme';
import SearchBooksApp from './SearchBooks.js';

const addBook = jest.fn();

it('renderiza corretamente', () => {
  expect(shallow(<SearchBooksApp addBook={addBook}/>)).toMatchSnapshot();
})
