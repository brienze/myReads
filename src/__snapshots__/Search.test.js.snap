// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderiza corretamente 1`] = `
<div
  className="search-books"
>
  <div
    className="search-books-bar"
  >
    <Link
      className="close-search"
      replace={false}
      to="/"
    >
      Close
    </Link>
    <div
      className="search-books-input-wrapper"
    >
      <input
        onChange={[Function]}
        placeholder="Search by title or author"
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="search-books-results"
  >
    <ol
      className="books-grid"
    />
  </div>
</div>
`;
